src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -55,8 +55,10 @@ enum class test_status
   warning = 1,
   failed = 2
 };
+enum class test_mode { support, json, no_json };
 
 constexpr uint64_t operator"" _gb(unsigned long long v)  { return 1024u * 1024u * 1024u * v; }
+static std::chrono::seconds MAX_TEST_DURATION(60 * 5); //5 minutes
```

Comment:
Hmm...this change doesn't seem to align with the pull request description. 
In addition, the variable name should indicate units (e.g., MAX_TEST_DURATION_SEC)